humility-core/src/core.rs

```diff
@@ -882,70 +984,38 @@ impl GDBCore {
             bail!("start/end inverted: \"{}\"", result);
         }
 
-        match result.find(GDB_PACKET_ACK) {
```

Comment:
I found that the gdb server never actually sent an ack back in this situation, and this would cause humility to hang.

Reply:
I see. As we discussed offline, I think it would be helpful to explain the reasoning behind the changes so that it will be easier to get the changes through review upstream.